sql/migrate_schema.sql

```diff
@@ -85,5 +85,9 @@ insert into hidden_packages values
 , ('sequin')
 on conflict do nothing;
 
+alter table packages
```

Comment:
I noticed that we already have a `links` column on the `packages` table.

However, given that `gleam_pgo` doesn't yet support JSON, it seemed more expedient to extract and store just the links we're interested in.

I think it's also a bit nicer from a database design perspective to have them in their own columns rather than in an untyped JSON bag.

Reply:
If we store links we should store all of them rather than just the two that are wanted right now. Happy to go with either JSON or another table. If a table we'll need to add some logic to remove outdated links also.

Reply:
> If we store links we should store all of them rather than just the two that are wanted right now.

What other links did you have in mind?

In [`hexpm.Package`](https://hexdocs.pm/gleam_hexpm/gleam/hexpm.html#Package) we also have `html_url`, but I don't think we need that one since we're building it dynamically.

Or are you referring just to the `links` field in [`hexpm.PackageMeta`](https://hexdocs.pm/gleam_hexpm/gleam/hexpm.html#PackageMeta)?

Reply:
Also, I've noticed that the `hexdocs.pm` links appear (?) to be deterministic:

```
https://hexdocs.pm/gleam_deno/
```

We might be able to avoid having to store the `html_docs_url` entirely and just build it dynamically using the package name (similar to what we do for the `html_url`).

Do you think it makes sense to do that vs storing what is returned from Hex?

We'll still have to store the other `links` in the `links` column, but I figured out how to get that working with the existing JSON column 🙂

Reply:
I guess one reason to use the `html_docs_url` from Hex is that there might be the possibility for a package to be published to Hex, but doesn't have any docs on Hexdocs?

In that case if we were to assume that the docs exist and link to them by constructing the link ourself, we could end up directing someone to a nonexistent page, which doesn't seem like a good user experience.

Reply:
> there might be the possibility for a package to be published to Hex, but doesn't have any docs on Hexdocs?

Yup, that's the one. Packages published by rebar3 or other tools might not have them